doc/index.md

```diff
@@ -14,7 +14,7 @@ See [Security](security.md) for more information.
 % Include content from [../README.md](../README.md)
 ```{include} ../README.md
     :start-after: security.md).
-    :end-before: greatly appreciated.
+    :end-before: Make sure to read
```

Comment:
I'm not familiar with this language, but is this including part of another file up to a particular starting sentence? If so, this feels like it would be quite brittle in the future, as people modifying the source file won't know the contents of the file is being used as an indicator for the basis of an include in another file.

Does the language support using an anchor or some other invisible tag that we can use as the indicator of a section ending?

Reply:
When including content from another file, we can specify the start and end with line numbers or with any string (see https://docutils.sourceforge.io/docs/ref/rst/directives.html#including-an-external-document-fragment ).
Using line numbers is very error-prone imo, so I usually go for strings.

As string, we can use pretty much anything, so for example also HTML comments (&lt;!-- something -->). However, that means we need to add the comment to the original file, which might be annoying when reading the file directly.

In some cases, using a comment is the only option (for example, if the content right after what we include is markup and not text), but if we can use text, I usually go for that. If someone adds a sentence right before, we probably want to include that in the other file as well. And if the "Make sure to read" sentence is removed, it'll raise a warning in the doc build.
But I'm absolutely fine with always going for comments, if we think that is less confusing.

Reply:
Up to @stgraber ofc, but I would prefer to have a comment in the source file which would serve both as an indicator in the source file that this section is embedded elsewhere, and as a less brittle way of specifying which sections to include.